src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Dynamic Sampling
+sidebar_order: 10
+description: "Learn about the different methods for controlling the amount and type of data sampled for your project."
+---
+
+<Note>
+
+This content is provided on a working branch as part of a project that is in progress. We anticipate it will be changed and modified as the project progresses.
```

Comment:
`changed` and `modified` is not redundant?

Reply:
yes it is, will change